htdocs/bvl_feedback_panel.class.inc

```diff
@@ -0,0 +1,58 @@
+ <?php
+/**
+ * General behavioural feedback panel class.
+ *
+ * This class provides the behavioural feedback gui for the timepoint list, instrument
+ * list and individual instruments.
+ *
+ * @package Behavioural
+ * @author Evan McIlroy <evanmcilroy@gmail.com>
+ * @license GNU
+ */
+class BVL_Feedback_Panel
+{
+    var $_feedbackThread = '';
+
+    function BVL_Feedback_Panel($candID, $sessionID=null, $commentID=null)
+    {
+        $user =& User::singleton();
+        $username = $user->getUsername();
+        $this->_feedbackThread =& NDB_BVL_Feedback::Singleton($username, $candID, $sessionID, $commentID);
+    }
+
+    function display(){
+        $this->tpl_data['candID'] = $this->_feedbackThread->_feedbackCandidateProfileInfo["CandID"];
+        $candidateID =  $this->_feedbackThread->_feedbackCandidateProfileInfo["CandID"];
+        $candidateObject = Candidate::singleton($candidateID);
+        $this->tpl_data['pscid'] = $candidateObject->getPSCID();
+        $this->tpl_data['sessionID'] = $this->_feedbackThread->_feedbackCandidateProfileInfo["SessionID"];
+        $this->tpl_data['commentID'] = $this->_feedbackThread->_feedbackObjectInfo["CommentID"];
+        $this->tpl_data['thread_list'] = $this->_feedbackThread->getThreadList();
+        $this->tpl_data['feedback_level'] = $this->_feedbackThread->_feedbackLevel;
+        $this->tpl_data['feedback_types'] = $this->_feedbackThread->getFeedbackTypes();
+        $this->tpl_data['feedbacktypes'] = NDB_BVL_Feedback::getFeedbackTypes();
+
+        $summary = $this->_feedbackThread->getSummaryOfThreads();
+        if (PEAR::isError($summary)) {
```

Comment:
We don't use PHP4 style exceptions anymore. getSummaryOfThreads should throw a (real) exception if applicable..